Sources/FigmaExport/Input/Params.swift

```diff
@@ -78,6 +78,9 @@ struct Params: Decodable {
             let swiftUIFontSwift: URL?
             let generateLabels: Bool
             let labelsDirectory: URL?
+            let nameValidateRegexp: String?
+            let nameReplaceRegexp: String?
+            let nameStyle: NameStyle
```

Comment:
These three parameters must be in the `Common` struct because someday I will make exporting typography for Android.

Reply:
Sorry. Only two parameters must be in the `Common` struct:
```
let nameValidateRegexp: String?
let nameReplaceRegexp: String?
```

Reply:
Sounds good. We're actually planning on implementing typography for Android soon to support our design system.